src/views/HomeView.vue

```diff
@@ -1,30 +1,13 @@
 <script setup>
-import { ref } from "vue";
+import { ref, watchEffect } from "vue";
 import Home from "../components/Home.vue";
+import { getTasks } from "../stores/api.js";
 
-const tasks = ref([
-  {
-    toolName: "Pywikibot",
-    toolDescription:
-      "Python library and collection of scripts that automate work on MediaWiki sites",
-    url: "https://www.mediawiki.org/wiki/Special:MyLanguage/Manual:Pywikibot",
-    missingField: "wikidata_qid",
-  },
-  {
-    toolName: "toolforge-authors",
-    toolDescription:
-      "This project is aimed at getting very basic author statistics for a specified page.",
-    url: "https://toolsadmin.wikimedia.org/tools/id/authors",
-    missingField: "openhub_id",
-  },
-  {
-    toolName: "toolforge-farotbot",
-    toolDescription:
-      "IABot fixes link rot replacing dead links with archive links, but it takes years to fully process the entire corpus of millions of articles. Featured articles require more frequent checks. This bot interfaces with IABot's API to trigger it to run on featured lists and articles on a regular basis.",
-    url: "https://toolsadmin.wikimedia.org/tools/id/farotbot",
-    missingField: "repository",
-  },
-]);
+const tasks = ref(null);
```

Comment:
Can we initialise tasks with an empty array instead of null.

Reply:
Hi @Damilare1 I had tried initializing an empty array for the tasks but it causes the logo to re-render with every reload in all the pages
<img width="1310" alt="image" src="https://user-images.githubusercontent.com/48224930/218691923-2fa51538-23a5-4f00-99fb-734cecd1c24e.png">

Reply:
Hmm weird behaviour, I just checked and noticed it may be related to this error on the console

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'tool')
Home.vue:74
```

This is because, when I added the optional chaining  on those nested fields (like `currentTask?.tool?.name`), it rendered only one of that container. I guess it's an issue with vuetify error handling. I had some comments about this in my earlier review. Adding those should fix it.

Reply:
works perfectly after adding the ternary null check. Thank you